src/test/functional/functional-tests.js

```diff
@@ -14,632 +14,686 @@
  * limitations under the License.
  */
 
-import { Client, Policy } from '../main/minio.js'
-import * as Minio from '../main/minio.js'
-import os from 'os'
-import stream from 'stream'
-import crypto from 'crypto'
-import async from 'async'
-import _ from 'lodash'
-import fs from 'fs'
-import http from 'http'
-import https from 'https'
-import url from 'url'
+const minio = require('minio');
+const Policy = minio.Policy;
+const Client = minio.Client;
+const os = require('os');
+const stream = require('stream');
+const crypto = require('crypto');
+const async = require('async');
+const _ = require('lodash');
+const fs = require('fs');
+const http = require('http');
+const https = require('https');
+const url = require('url');
+//const chai = require('chai');
 import { assert } from 'chai'
-import superagent from 'superagent'
+const superagent = require('superagent');
+var uuid = require("uuid");
```

Comment:
is there a reason to move to this style of package imports? due to avoiding ES6 transpiling stage?